TASVideos/TASVideos.csproj

```diff
@@ -56,5 +56,6 @@
 
 	<ItemGroup>
 		<ProjectReference Include="..\TASVideos.Api\TASVideos.Api.csproj" />
+		<InternalsVisibleTo Include="TASVideos.RazorPages.Tests" />
```

Comment:
This one isn't working ~~for some reason~~ because of `<GenerateAssemblyInfo>false</GenerateAssemblyInfo>` above. I'll leave it as `AssemblyInfo.cs`.